src/components/app-login/app-login.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component, h, Method, Prop, State } from '@stencil/core';
+
+@Component({
+  tag: 'app-login',
+  styleUrl: 'app-login.css',
+  shadow: true,
+})
+
+
+
+export class AppLogin {
+
+  @Prop() loading: boolean = false;
```

Comment:
boolean can be remove, as typescript can infer types automatically when initiallised